### ISSUE TITLE
training with no effect?

### ISSUE THREAD

**Issue by delmenhorst** [d8ahazard/sd_dreambooth_extension#685]
Hello,
I have managed to train Dreambooth  on 10GB VRAM . Unfortunately, the training has no effect on the models.

I follow the usual settings, used 10~20 training images and roughly 100 class images per image. Regardless of whether I use the Dreambooth extension or ShivamShrirao Dreambooth code, there are no useful results? 

usual settings : 
1000 to 2000cpkt 
"instance_prompt":      "photo of zwx person",
"class_prompt":         "photo of a person";

All paths are also correct, the configuration of the training seems to be correct. 
When I try to generate an image with the instance prompt, the result has nothing to do with my target. 
I use the same scheduler when generating. 

Is there anything else that I am doing wrong? Do you need to pay attention when saving?

```
---
Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0]
Commit hash: c0ee1488702d5a6ae35fbf7e0422f9f685394920
Installing requirements for Web UI

#######################################################################################
Initializing Dreambooth
If submitting an issue on github, please provide the below text for debugging purposes:

Python revision: 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0]
Dreambooth revision: 4ca69a904f5ddd5651d87032b3dca515eea505ba
SD-WebUI revision: c0ee1488702d5a6ae35fbf7e0422f9f685394920

Checking Dreambooth requirements...
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] xformers version 0.0.15+e163309.d20230103 installed.
[+] torch version 1.13.1 installed.
[+] torchvision version 0.14.1 installed.

---
OS: Ubuntu 22.04.1 LTS x86_64
CPU: AMD Ryzen 9 5900X (24) @ 3.700G
GPU: NVIDIA GeForce RTX 3080 
NVIDIA-SMI 515.86.01    
Driver Version: 515.86.01    
CUDA Version: 11.7


```

**Comment by vrgz2022** [comment]
I used 700 pictures to train the style, but there was no response. It was normal for me to use notebook to train the same dataset on the Ubuntu system. At one time, I thought it was my own setting problem, but now it is not my problem

**Comment by FurkanGozukara** [comment]
ye because it is very hard to get correct configuration. i finally managed will publish a video

**Comment by alenknight** [comment]
> 

i'd be interested in this too.  with regular SD 1.5 i can get results.  but when i try to train custom ckpt... like analog diffusion... it seems to have no effect.

**Comment by lugia19** [comment]
Lora is completely nonfunctional on the latest few commits. You should revert back to commit fab41d85da47bc2901a1db712e8ae265306c93b9. You'll notice if you've got this issue because when you export the model as ckpt it won't have the _lora suffix.

**Comment by ciprian-hetea** [comment]
You can train on the SD 1.5 model and then merge it with a custom ckpt. It's not going to be exactly the same thing, but it'll get you pretty close.

**Comment by d8ahazard** [comment]
Fixed with eb51824.
